Cabal/doc/nix-local-build.rst

```diff
@@ -118,9 +118,9 @@ across projects. To be more precise:
 1. A **local package** is one that is listed explicitly in the
    ``packages``, ``optional-packages`` or ``extra-packages`` field of a
    project. Usually, these refer to packages whose source code lives
-   directly in a folder in your project (although, you can list an
```

Comment:
I find the notes in parenthesis everywhere a bit annoying (forcing me to context-switch :sweat_smile: , breaking the flow of text)